cmd/ping.go

```diff
@@ -0,0 +1,113 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/heartbeatctl/pkg/client"
+	"github.com/giantswarm/heartbeatctl/pkg/cmdutil"
+	"github.com/giantswarm/heartbeatctl/pkg/ctl"
+)
+
+// pingCmdOptions holds values for options accepted by the ping command
+type pingCmdOptions struct {
+	selectorOptions *cmdutil.SelectorOptions
+}
+
+var (
+	pingDocLong = heredoc.Doc(`
+	    Issue a ping request to specified heartbeats.
+		Beware that receiving a successful response does not necessarily mean that 
+		the heartbeat exists: https://docs.opsgenie.com/docs/heartbeat-api#ping-heartbeat-request
+		Heartbeats to ping can be specified by a combination of selectors all of
```

Comment:
Indentation feels wrong and we can use more spaces here.

```suggestion
	        Issue a ping request to specified heartbeats.

		Beware that receiving a successful response does not necessarily mean that 
		the heartbeat exists: https://docs.opsgenie.com/docs/heartbeat-api#ping-heartbeat-request

		Heartbeats to ping can be specified by a combination of selectors all of
```

Reply:
Sorry got confuse this is actually for https://github.com/giantswarm/heartbeatctl/pull/15/

Reply:
Solved in 15